docs/meshstack.managed-service.md

```diff
@@ -29,7 +29,7 @@ meshcloud provides customers with comprehensive security and compliance document
 
 ## Continuous Delivery
 
-meshcloud releases new versions of meshStack on a continuous schedule. As you can see by reviewing our [release notes](/blog), this can happen multiple times a week. Our managed service ensures that your installation and configuration is kept up to date at all times when delivering new releases to your staging and production environments.
+meshcloud releases new versions of meshStack on a continuous schedule. As you can see by reviewing our [release notes](/blog), this can happen multiple times a week. Our managed service ensures that your installation and configuration is kept up to date at all times when delivering new releases to your [staging](#staging-environment) and [production environments](#production-environment).
```

Comment:
```suggestion
meshcloud releases new versions of meshStack on a continuous schedule. As you can see by reviewing our [release notes](/blog), this can happen multiple times per week. Our managed service ensures that your installation and configuration is up to date at all times when delivering new releases to your [staging](#staging-environment) and [production environments](#production-environment).
```